app/views/boards/_reorder_posts.haml

```diff
@@ -12,7 +15,7 @@
       Saved
   %ul.sortable.table-list
     - unsectioned_posts.each do |post|
-      %li.section-ordered{class: cycle('even', 'odd'), data: { id: post.send(local_assigns[:id] || :id), order: post.section_order }}
+      %li.section-ordered{class: cycle('even', 'odd'), data: { id: post.send(id || :id), order: post.section_order }}
```

Comment:
(could probably combine this into `id: :id` above instead?)

Reply:
huh, does that work? i'm kind of confused on what this is doing tbh

Reply:
It should set `:id` as the default value for the `id` field, which is what this was doing before - `local_assigns[:id]` if it's not falsy (nil/false), otherwise `:id`

Assuming nothing's explicitly passing in `id: nil`, but instead `id: :some_other_value`, it should be equivalent

Reply:
oh well then we probably just want the default to be -- ah, :id, yes